pipeline.groovy

```diff
@@ -179,6 +179,12 @@ def testDownstreamProject (name) {
 *******************************************************************************/
 
 node { // for now whole pipeline runs on one node because no slaves are present
+    /* Use the same workspace, no matter what job (dmd, druntime,...)  triggered
+     * the build.  The workspace step will take care of concurrent test-runs and
+     * allocate additional workspaces if necessary.  This setup avoids to
+     * reclone repos for each test-run.
+     */
+    ws: 'dlangci'
```

Comment:
It's based on the job name (see https://github.com/jenkinsci/workflow-durable-task-step-plugin/blob/8f7dac968c97e2a123befc2419f16e24234ca785/src/main/java/org/jenkinsci/plugins/workflow/support/steps/WorkspaceStepExecution.java#L45-L58).
`ls /var/lib/jenkins/workspace` @ runner-01.ci2.lxd
```
-test-organization_dmd_PR-2-43RSTKUS5XTSIZNXNWDK5S56E547FOJ4WDFVZTW5CQE7DI65LTJQ
-test-organization_dmd_PR-2-43RSTKUS5XTSIZNXNWDK5S56E547FOJ4WDFVZTW5CQE7DI65LTJQ@2
-test-organization_dmd_PR-2-43RSTKUS5XTSIZNXNWDK5S56E547FOJ4WDFVZTW5CQE7DI65LTJQ@2@tmp
-test-organization_dmd_PR-2-43RSTKUS5XTSIZNXNWDK5S56E547FOJ4WDFVZTW5CQE7DI65LTJQ@3
-test-organization_dmd_PR-2-43RSTKUS5XTSIZNXNWDK5S56E547FOJ4WDFVZTW5CQE7DI65LTJQ@4
-test-organization_dmd_PR-3-GOQOXUB3V5PYYBYOIE2XZ3DSWD4IWUVUIOVANGFNIK7FT3OXDKRQ
-test-organization_dmd_PR-3-GOQOXUB3V5PYYBYOIE2XZ3DSWD4IWUVUIOVANGFNIK7FT3OXDKRQ@tmp
```